api/api/openapi.bundle.yaml

```diff
@@ -1441,6 +1512,64 @@ components:
     Id:
       format: int32
       type: integer
+    EnsemblerImages:
+      items:
+        $ref: '#/components/schemas/EnsemblerImage'
+      type: array
+    EnsemblerImage:
+      example:
+        image_ref: image_ref
+        project_id: 0
+        image_building_job_status:
+          message: message
+        exists: true
+        ensembler_id: 6
```

Comment:
Wow there's even an example generated. I wonder if this was generated by the openapitools form 🤔